src/firmware-update.html

```diff
@@ -0,0 +1,28 @@
+---
+layout: base
+description: Zigbee and Thread USB stick by the creators of Home Assistant
+---
+<div class="banner">
+  <span class="secondary">Web-Flasher</span>
+  <span>Firmware Updates for SkyConnect</span>
+</div>
+<div class="container">
+  <div>
+    <h1>Firmware update</h1>
+    <p>First things first: a firmware upgrade is not required for Home Assistant SkyConnect at this point.</p>
```

Comment:
"First things first" (to me) sounds unusually informal IMO:

```suggestion
    <p>A firmware upgrade is not required for the Home Assistant SkyConnect at this point. There are no known bugs with the firmware it ships with.</p>
```

Reply:
Thank you @puddly!